fix(ci): GitHub Actions iOS関連ワークフローの修正

## 概要

PR #81 のCIで発見されたGitHub Actions iOS関連ワークフローの設定問題を修正する。

**注意**: これらはワークフロー設定の問題であり、ソースコードの品質問題ではない。

---

## 🔴 修正が必要なワークフロー

### 1. SwiftLint (`swiftlint.yml`)

**問題**: `unzip`コマンドでLICENSEファイル上書き確認が発生し、インタラクティブプロンプトで停止

**エラーログ**:
```
replace LICENSE? [y]es, [n]o, [A]ll, [N]one, [r]ename:
```

**修正案**:
```yaml
# Before
- run: unzip swiftlint.zip

# After
- run: unzip -o swiftlint.zip
```

---

### 2. Dependency Audit (`ios-quality-gate.yml`)

**問題**: 存在しない `cocoapods-audit` gem を参照

**エラーログ**:
```
ERROR:  Could not find a valid gem 'cocoapods-audit' (>= 0) in any repository
```

**修正案**:
- `cocoapods-audit` の参照を削除
- または別のiOS依存関係監査ツールに置き換え

---

### 3. iOS Build (`ios-quality-gate.yml`)

**問題**: `xcodebuild` が exit code 65 で失敗

**エラーログ**:
```
xcodebuild: error: ...
** BUILD FAILED **
Command exited with code 65
```

**修正案**:
- macOSランナーの設定を見直し
- Xcodeバージョンの明示的な指定
- ワークスペース/スキーム設定の確認

---

### 4. build-metrics (`ios-build-metrics.yml`)

**問題**: 同様に `cocoapods-audit` の依存関係問題

**修正案**: Dependency Auditと同様の修正

---

### 5. dependency-check (`security-scan.yml`)

**問題**: iOS関連の依存関係チェックが失敗

**修正案**: iOS固有のチェックを分離または無効化

---

## 📋 タスクリスト

- [ ] SwiftLint: `unzip -o` オプション追加
- [ ] Dependency Audit: `cocoapods-audit` 参照を削除
- [ ] iOS Build: macOSランナー設定の見直し
- [ ] build-metrics: 依存関係修正
- [ ] dependency-check: iOS固有チェックの分離

---

## 🔗 関連

- PR #81 (マージ済み): Xcode Cloudビルド対応
- Xcode Cloud: 正常動作確認済み（GitHub Actionsとは独立）

---

## 優先度

**Medium** - Xcode Cloudでのビルドは正常動作するため、GitHub ActionsのiOS CIは補助的な役割

--- a/.github/workflows/lint.yml
+++ b/.github/workflows/lint.yml
@@ -33,7 +33,7 @@ jobs:
           # SwiftLint の最新版をダウンロード
           SWIFTLINT_VERSION="0.58.0"
           curl -sL "https://github.com/realm/SwiftLint/releases/download/${SWIFTLINT_VERSION}/swiftlint_linux.zip" -o swiftlint.zip
-          unzip -q swiftlint.zip
+          unzip -o -q swiftlint.zip -x LICENSE
           sudo mv swiftlint /usr/local/bin/
           swiftlint version
 

--- a/.github/workflows/security-scan.yml
+++ b/.github/workflows/security-scan.yml
@@ -35,7 +35,7 @@ jobs:
       - name: Run OWASP Dependency-Check
         uses: dependency-check/Dependency-Check_Action@main
         with:
-          project: "claude-code-template-mcp"
+          project: "final-hourglass"
           path: "."
           format: "ALL"
           args: >

